Octokit.fsx

```diff
@@ -0,0 +1,80 @@
+#I __SOURCE_DIRECTORY__
+#r "./Octokit/bin/Release/Net45/Octokit.dll"
+
+open Octokit
+open System
+open System.IO
+
+type Draft =
+    { Client : GitHubClient
+      Owner : string
+      Project : string
+      DraftRelease : Release }
+
+let private isRunningOnMono = System.Type.GetType ("Mono.Runtime") <> null
+
+let rec private retry count asyncF =
+    // This retry logic causes an exception on Mono:
```

Comment:
`isRunningOnMono` is used for `SourceLink` . Sourcelink is excluded on mono https://github.com/fsharp/FAKE/blob/master/build.fsx#L304